src/main/java/com/ithaca/message/Message.java

```diff
@@ -0,0 +1,71 @@
+package com.ithaca.message;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.ithaca.user.User;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Created by David on 4/4/16.
```

Comment:
Shelby and Chris should be looking through this... They should be looking at your code so they have an idea whats happening, and to learn about code reviews.